Rakefile

```diff
@@ -47,6 +47,14 @@ def search_server
   search_config.search_server
 end
 
+def bucket
```

Comment:
The bucket definitely can.

elasticsearch_uri is also used by indices.rake, because I've added a new index related task that uses the elasticsearch client.

I put the implementation in index.rb since it's related to the other functionality there, although maybe there is somewhere else to put this. The existing code always accesses that url indirectly via the SearchServer instance, but for this we don't need to instantiate the whole hierarchy of SearchServer, IndexGroup, Index for anything else, so it was easier to just make it a class method and pass in the url config.